.travis_dev.yml

```diff
@@ -0,0 +1,25 @@
+language: node_js
+node_js: "8"
+script:
+  - yarn lint
+  - yarn test
+  - if [ "$TRAVIS_PULL_REQUEST" = "false" ] && ([ "$TRAVIS_BRANCH" == "master" ] || [ "$TRAVIS_BRANCH" == "staging" ]); then yarn run build; fi
+after_success:
+  - bash <(curl -s https://codecov.io/bash)
+cache:
+  yarn: true
+  directories:
+  - node_modules
+deploy:
+  - provider: s3
+    access_key_id: $AWS_ACCESS_KEY_ID
```

Comment:
If we're deploying to the new AWS account for EP vs the `cruds` account, these two values are probably wrong.

Reply:
@wrandall22 : Couldn't configure the AWS Access and Secret keys as these details has to be configured in the Environmental  variable of build settings in travis UI [or in the server where travis is installed as my environment specific changes couldn't be replicated by travis deploy]. Please help in configuring the same and share variable details. So that I would submit the PR for the same.

Reply:
I created `$8_1_AWS_ACCESS_KEY_ID` and `$8_1_AWS_SECRET_ACCESS_KEY` with the values you shared with me.